GW2EIEvtcParser/EIData/ProfHelpers/Necromancer/NecromancerHelper.cs

```diff
@@ -314,7 +314,7 @@ internal static void ComputeProfessionCombatReplayActors(AbstractPlayer player,
             // Reaper's Mark (Staff 5)
             if (log.CombatData.TryGetEffectEventsBySrcWithGUID(player.AgentItem, EffectGUIDs.NecromancerReapersMark, out IReadOnlyList<EffectEvent> reapersMarks))
             {
-                var skill = new SkillModeDescriptor(player, Spec.Necromancer, ReapersMark);
+                var skill = new SkillModeDescriptor(player, Spec.Necromancer, ReapersMark, SkillModeCategory.CC);
```

Comment:
Is the category needed for the non activated version? For example we don't add "Portal" category to non opened portal skills.